src/js/components/DataTable/stories/typescript/Clickable.tsx

```diff
@@ -140,19 +140,43 @@ const DATA: RowType[] = [
   },
 ];
 
-export const ClickableDataTable = () => (
-  <Grommet theme={grommet}>
-    <Box align="center" pad="large">
-      {/* eslint-disable no-alert */}
-      <DataTable
-        columns={columns}
-        data={DATA}
-        step={10}
-        onClickRow={event => console.log(event.datum)}
-      />
-    </Box>
-  </Grommet>
-);
+export const ClickableDataTable = () => {
+  const [show, setShow] = React.useState(false);
+  const [clicked, setClicked] = React.useState({});
+
+  return (
+    <Grommet theme={grommet}>
+      <Box align="center" pad="large">
+        {/* eslint-disable no-alert */}
```

Comment:
why do we need this eslint-disable?

Reply:
This will be trimmed in the next commit, totally forgot to get rid of this as the first iteration of this story was using an `alert()`